### PR TITLE
Add label for resources create by EventListener

### PR DESCRIPTION
**Pull request by vincent-pli**
Fix the issue: #48 
If resources already has `ownerReference`, will not add `EventListener` as it's owner. Since we suppose the resource has it's own garbage collect mechanism, we will not broke it.

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/triggers/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

```
Describe any user facing changes here, or delete this block.

Examples of user facing changes:
- API changes
- Bug fixes
- Any changes in behavior

```

### CODE REVIEW

**Comment by vtereso** [comment]
@vincent-pli Looks good. We discussed this in the WG (notes [here](https://docs.google.com/document/d/1T87yK4BIu291gGK1L2ZzDpesGCnXX3tGuWXjdr5Soxw/edit#heading=h.b7qqx1bek9fi)), but I don't think we have come to a conclusion yet whether we want to add OwnerReferences on the generated resources or if https://github.com/tektoncd/triggers/issues/48 would be enough.

File: pkg/sink/sink.go [hunk 1]
```diff
@@ -94,6 +107,11 @@ func createResource(rt json.RawMessage, restClient restclient.Interface, discove
 	if err != nil {
 		return err
 	}
+
+	if !gjson.GetBytes(rt, "metadata.ownerReferences").Exists() {
+		rt, _ = sjson.SetBytes(rt, "metadata.ownerReferences", ownerRef)
```
Comment (vtereso) [inline pkg/sink/sink.go]: So if there were owners, this would not add the `EventListener`? Makes sense.

**Comment by vincent-pli** [comment]
@vtereso Thanks for comments. Actually, I'm in the WG meeting last night, you guys speak so fast 😭 , pain of non-native speaker!!!

Whatever, i prefer add `ownerReferences`,  I think we need supply approach to clear resource easier.
As i mentioned in #64 , it's a problem we need consider if expect adopte `Trigger` in product env.
if there is other approach, I'm glad to follow.

**Comment by vincent-pli** [comment]
As the discussion here, firstly add the label to resources.
https://github.com/tektoncd/triggers/issues/64#issuecomment-529946277

**Comment by vtereso** [comment]
My fault for not recognizing the commit message hadn't been updating to reflect the new title :/ This PR does NOT 'Add ownerReference for resources create by EventListener', but instead does as declared in the title (adds labels for resources generated by `EventListener`).

**Pull request merged**